### ISSUE TITLE
Starfails parses aegea batch ls incorrectly to find failed jobs

### ISSUE THREAD

**Issue by shayanhoss** [czbiohub/utilities#30]
```(utilities-env) shayanhoss@ndnd:~/maca_realign_jobs$ starfails 180626_A00111_0166_BH5LNVDSXX_failed_jobs.sh 
??aafa57c3-91b1-4dea-a530-9a6af1a861
Traceback (most recent call last):
  File "/home/shayanhoss/miniconda3/envs/utilities-env/bin/aegea", line 23, in <module>
    aegea.main()
  File "/home/shayanhoss/miniconda3/envs/utilities-env/lib/python3.6/site-packages/aegea/__init__.py", line 82, in main
    result = parsed_args.entry_point(parsed_args)
  File "/home/shayanhoss/miniconda3/envs/utilities-env/lib/python3.6/site-packages/aegea/batch.py", line 452, in describe
    return clients.batch.describe_jobs(jobs=[args.job_id])["jobs"][0]
  File "/home/shayanhoss/miniconda3/envs/utilities-env/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/shayanhoss/miniconda3/envs/utilities-env/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.ClientException: An error occurred (ClientException) when calling the DescribeJobs operation: Provided name \udc94\udc82aafa57c3-91b1-4dea-a530-9a6af1a861 is not supported pattern[a-zA-Z_0-9-]{1,128}```

**Comment by shayanhoss** [comment]
```(utilities-env) shayanhoss@ndnd:~/maca_realign_jobs$ aegea batch ls --status FAILED -c jobId | grep "minutes ago" 
│a3ce933c-edf3-4d69-afe0-167d0ef63095│42 minutes ago│
│422bed55-99af-491d-a6da-b35cf624b896│29 minutes ago│```

Parsing at 2-37 and should be between 4-39 to grab full `job_id`

**Comment by shayanhoss** [comment]
Parsing at 2-37 and should be between 4-39 to grab full `job_id`

**Comment by jamestwebber** [comment]
I think this was fixed by #31 because the starfails script is now using the aegea API.
